client/js/components/GameView/Card.vue

```diff
@@ -134,6 +141,26 @@ export default {
     isBack() {
       return !this.suit && !this.rank;
     },
+    scuttledByTransition() {
+      switch (this.controlledBy) {
+        case 'player':
+          return 'slide-above';
+        case 'opponent':
+          return 'slide-below';
+        default:
+          return '';
+      }
+    },
+    scuttledByClass() {
+      switch (this.controlledBy) {
+        case 'player':
+          return 'scuttled-by-card scuttled-by-opponent';
+        case 'opponent':
+          return 'scuttled-by-card scuttled-by-player';
+        default:
+          return '';
```

Comment:
Is there a case where this is ever empty?

Reply:
A few places:

1. Cards in the deck during 7 resolution
2. Cards in dialogs e.g. the scrap, 3 resolution, 4 resolution, countering
3. Arguable whether it's relevant for displaying the selected card in the MoveChoiceOverlay

I currently haven't applied the prop to any cards except player points and opponent points because those were the only ones using it, but it could additionally be put on face cards and jacks